vector/src/main/res/layout/fragment_ftue_server_selection_combined.xml

```diff
@@ -120,7 +120,7 @@
             app:layout_constraintHeight_percent="0.03"
             app:layout_constraintTop_toBottomOf="@id/chooseServerInput" />
 
-        <Button
+        <com.google.android.material.button.MaterialButton
```

Comment:
A bit annoying that we have to do this change. I think `MaterialButton` will be inflated when declaring `Button` in the layout.

Reply:
yeah there's something strange going on with the theming and default button with paparazzi, the `background hint` ends up being ignored :shrug: